reference/orders.v2.oas2.yml

```diff
@@ -444,10 +531,31 @@ paths:
                           value: '180'
                         - id: 230
                           value: '192'
+              Adding a fee to the order (fee provided without an ID):
+                value:
+                  fees:
+                    - type: custom_fee
+                      display_name_customer: D Fee
+                      source: v2
+                      cost_ex_tax: '22.5000'
+                      cost_inc_tax: '25.0000'
+                      tax_class_id: 22
+                    - type: custom_fee
+                      display_name_customer: E Fee
+                      source: v2
+                      cost_exc_tax: '12.3000'
+                      cost_inc_tax: '12.30000'
+                      tax_class_id: 22
+                    - type: custom_fee
+                      display_name_merchant: F Fee
+                      source: v2
+                      cost_ex_tax: '4.0000'
+                      cost_tax: '8.0000'
+                      tax_class_id: 0
```

Comment:
seems like this payload is not enough for POST /orders request, probably we can remove this example ?